src/Text/Pandoc/Readers/Org/Blocks.hs

```diff
@@ -418,12 +418,15 @@ orgParamValue :: Monad m => OrgParser m Text
 orgParamValue = try $ fmap T.pack $
   skipSpaces
     *> notFollowedBy orgArgKey
-    *> noneOf "\n\r" `many1Till` endOfValue
+    *> (quotedValue <|> regularValue)
     <* skipSpaces
  where
   endOfValue = lookAhead $  try (skipSpaces <* oneOf "\n\r")
                         <|> try (skipSpaces1 <* orgArgKey)
 
+  quotedValue = char '\"' *> manyTill anyChar (char '\"')
+
+  regularValue = noneOf "\n\r" `many1Till` endOfValue
```

Comment:
This will parse spaces; is that intended?
The old parser would stop if it gets to spaces followed by an orgArgKey, while this one won't, if I understand correctly.